src/main/scala/fr/polytechnique/cmap/cnam/etl/transformers/exposures/ExposureDuration.scala

```diff
@@ -48,6 +48,8 @@ case class ExposureDuration(patientID: String, value: String, period: Period, sp
               RightRemainingPeriod(ExposureDuration(self.patientID, self.value, p2, self.span))
             )
           }
+        // avoid scala match may not be exhaustive
```

Comment:
What is the impact of this clause ? What is the behaviour when this happens?

Reply:
this clause avoids a warning when compiling the code of match pattern